spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe ItemsController do
+
+  after(:each) do
+    Item.destroy_all
```

Comment:
Database Cleaner can take care of this for you: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/support/database_cleaner.rb